### PR TITLE
add keystore.md

### PR DESCRIPTION
**Pull request by spetrychenko**

### CODE REVIEW

File: docs/keystore.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+## Adapter keystore
+Adapter uses keystore (PKCS #12 **java.security.KeyStore**) for holding certificates and keys which are used for signing http messages (QSEAL) and client tls authentication (QWAC).
+
+By default, a keystore must contain two bags named "default_qwac" and "default_qseal" with an empty password.
```
Comment (berezkin88) [inline docs/keystore.md:4]: I think it's better to use the same description as in `Pkcs12KeyStore.class` provided - "a keystore is expected to have/include ...". 
In this variant, it sounds like an obligation to have `two bags named "default_qwac" and "default_qseal"`, which is not true as there is a way for a user to provide his/her own aliases for QWAC and QSEAL

File: docs/keystore.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+First create a p12 file for each certificate/key pair with a specific alias.
+```
+openssl pkcs12 -export -out <p12_file> -in <cert_file> -inkey <key_file> -name <alias>
+```
+And then combine all p12 files into one.
+```
+keytool -importkeystore -srckeystore <src_p12> -destkeystore <dest_p12> -srcstorepass '' -deststorepass ''
+```
+You can specify the path to your keystore with a java vm option **pkcs12.keyStore**. E.g.
```
Comment (berezkin88) [inline docs/keystore.md:15]: You have written the word in two different ways: `key store` and `keystore`. Please stick to one particular option.

File: docs/keystore.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+```
+java -Dpkcs12.keyStore=/path/to/your/keystore.p12 ...
+```
```
Comment (berezkin88) [inline docs/keystore.md:18]: Probably it's worth mentioning an option for specifying a KeyStore password if a user has one? e.g. `-Dpkcs12.keyStorePassword=<keyStorePassword>`

File: docs/keystore.md (hunk 1 above)
Comment (spetrychenko) [reply docs/keystore.md]: adjusted

File: docs/keystore.md (hunk 2 above)
Comment (spetrychenko) [reply docs/keystore.md:15]: fixed

File: docs/keystore.md (hunk 3 above)
Comment (spetrychenko) [reply docs/keystore.md:18]: added, thanks

**Review by berezkin88 — APPROVED** [review]

**Pull request merged**